### PR TITLE
[RTL] Fix main table and overlays misalignment

### LINKED ISSUE #8761: [RTL] BottomOverlay
### Description

When I set `dir='rtl'` the bottom overlay displayed incorrectly on the left of the website. It should display on the right, below the table.

### Steps to reproduce

Following this [configuration](https://github.com/Siemienik/handsontable-storybooks/blob/10bd304f4ae389538f62abc136a1cb531f5a4acd/vanilla/configurations.js#L436-L440) 

```js
configurations.FixedRowsBottom = {
    data: Handsontable.helper.createSpreadsheetData(60, 10),
    licenseKey: "non-commercial-and-evaluation",
    fixedRowsBottom: 1
};
```
I got: 
image

### PR DESCRIPTION
**Pull request by budnix**
### Context

The PR fixes the overlays misalignment that happens in the RTL document mode.

To emphasize the overlays I colorize them within the gifs.

#### Case where HoT is initialized within `overflow: hidden` DOM element.

_The same case is when the configuration contains `width` and `height` options._

<details>
  <summary>Configuration</summary>

To the `#example` element these CSS props should be assigned.
```css
#example {
  overflow: hidden;
  height: 250px;
  width: 500px;
}
```
```js
  new Handsontable(document.querySelector('#example'), {
    licenseKey: 'non-commercial-and-evaluation',
    data: Handsontable.helper.createSpreadsheetData(500, 200),
    rowHeaders: true,
    colHeaders: true,
    fixedColumnsStart: 2,
    fixedRowsBottom: 2,
    fixedRowsTop: 2,
  });
```
  </details>

LTR mode:
Kapture 2022-01-11 at 15 06 56

RTL mode:
Kapture 2022-01-11 at 15 12 04

#### Case where HoT is initialized without defined dimensions.

_The case where the scrollable element is the Window object._

<details>
  <summary>Configuration</summary>

The `#example` element should not contain any CSS properties.
```js
  new Handsontable(document.querySelector('#example'), {
    licenseKey: 'non-commercial-and-evaluation',
    data: Handsontable.helper.createSpreadsheetData(500, 200),
    rowHeaders: true,
    colHeaders: true,
    fixedColumnsStart: 2,
    fixedRowsBottom: 2,
    fixedRowsTop: 2,
  });
```
  </details>

LTR mode:
Kapture 2022-01-11 at 15 15 51

RTL mode:
Kapture 2022-01-11 at 15 16 58

_[skip changelog]_

### How has this been tested?

I tested the changes locally and cover the fixes with new E2E tests.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

### Related issue(s):
1. #8760 
2. resolves  #8761
3. resolves: #8762
4. resolves: #8763
5. resolves: #8764
6. resolves: #8765
7. resolves: #9094

### Affected project(s):
- [x] `handsontable`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [x] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)

### CODE REVIEW

**Comment by Siemienik** [comment]
Review todo:
* [ ] https://github.com/handsontable/handsontable/blob/4e8df3298bf0338b6c3ff390b419ffbef99f2a92/handsontable/src/editors/textEditor/textEditor.js#L367-L385

**Review by Siemienik — COMMENTED** [review]
Wow! you did it excellent! I reviewed the code, and I tested the local runtime. It looks good to me. 

I see only one issue with CSS properties naming, which I want to put to the board at the team chat or meeting.
image

In RTL, the start overlays (on the right-hand side) have names `left`
image

With full respect to the datelines, I would like only to point that one introduced name for CSS class is hard to change later and always is tracked as breaking change. In my human opinion, it is better to introduce new names: `ht_clone_right`, `ht_clone_top_right_corner`, `ht_clone_bottom_right_corner`. I suppose that might be time-consuming to do.

**Review by Siemienik — APPROVED** [review]
According to our meeting, I approve these changes in the current shape to prevent further blockers. The naming scheming will be discussed and finally approved at the next meeting, then I'm expecting another PR with applying decisions.

Thank you for the effort, LGTM 👏

File: handsontable/src/3rdparty/walkontable/src/overlay/constants.js [hunk 1]
```diff
@@ -3,14 +3,13 @@
  */
 export const CLONE_TOP = 'top';
 export const CLONE_BOTTOM = 'bottom';
-export const CLONE_LEFT = 'left';
-export const CLONE_TOP_LEFT_CORNER = 'top_left_corner';
-export const CLONE_BOTTOM_LEFT_CORNER = 'bottom_left_corner';
+export const CLONE_INLINE_START = 'left';
+export const CLONE_TOP_INLINE_START_CORNER = 'top_left_corner';
+export const CLONE_BOTTOM_INLINE_START_CORNER = 'bottom_left_corner';
 export const CLONE_TYPES = [
```
[the remark this answers is not in the document]
Comment (Siemienik) [reply handsontable/src/3rdparty/walkontable/src/overlay/constants.js:9]: see https://github.com/handsontable/handsontable/pull/9098#pullrequestreview-853272449

**Comment by budnix** [comment]
> According to our meeting, I approve these changes in the current shape to prevent further blockers. The naming scheming will be discussed and finally approved at the next meeting, then I'm expecting another PR with applying decisions

#9130

**Pull request merged**